mat64/matrix.go

```diff
@@ -409,7 +409,7 @@ func Dot(a, b Matrix) float64 {
 	return sum
 }
 
-// Equal returns whether element a and b have the same size and contain all equal
+// Equal returns whether the matrices a and b have the same size and contain all equal
```

Comment:
What would be a better wording?

```
Equal returns whether the matrices a and be have the same size
and are element-wise equal.
```
? Also maybe I prefer to replace `size` with `shape`.